src/ToyStorage.UnitTests/InMemoryResponseCacheMiddlewareTests.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ToyStorage.UnitTests
+{
+    public class InMemoryResponseCacheMiddlewareTests
+    {
+        [Fact]
+        public async Task TestGetGet()
```

Comment:
tests kind of suck. just checking that it doesn't break. would be useful to do some unit tests to actually validate its going to cache when appropriate.